<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InoSettings</class>
 <widget class="QDialog" name="InoSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>160</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>LSYNC PW:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLsyncPw">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLsyncPwGet">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Get</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLsyncPwSet">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonComputeOffset">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Compute Offset</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFpaTemperature">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>120</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>FPA Temperature:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLoadRpl">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load RPL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditShcdsPw">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonShcdsPwSet">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonShcdsPwGet">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Get</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>200</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SH_CDS PW:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditShcdsStart">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>230</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SH_CDS Start:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditShvidPw">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonShvidPwSet">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonShvidPwGet">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Get</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>270</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SH_VID PW:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditShvidStart">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>300</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SH_VID Start:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditVresHigh">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFpaSet">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>220</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFpaGet">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Get</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>VRES_HIGH:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>FPA BIAS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>VCSH:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditVcsh">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>150</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>VCSL:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditVcsl">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>VREF:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditVref">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>210</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>VCM1:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditVcm1">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>240</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>VCM2:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditVcm2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>VCM3:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditVcm3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>300</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFpaTemperatureGet">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Get</string>
   </property>
  </widget>
  <zorder>pushButtonOK</zorder>
  <zorder>label</zorder>
  <zorder>pushButtonLsyncPwGet</zorder>
  <zorder>pushButtonLsyncPwSet</zorder>
  <zorder>pushButtonComputeOffset</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButtonLoadRpl</zorder>
  <zorder>lineEditShcdsPw</zorder>
  <zorder>pushButtonShcdsPwSet</zorder>
  <zorder>pushButtonShcdsPwGet</zorder>
  <zorder>label_3</zorder>
  <zorder>lineEditShcdsStart</zorder>
  <zorder>label_4</zorder>
  <zorder>lineEditShvidPw</zorder>
  <zorder>pushButtonShvidPwSet</zorder>
  <zorder>pushButtonShvidPwGet</zorder>
  <zorder>label_5</zorder>
  <zorder>lineEditShvidStart</zorder>
  <zorder>label_6</zorder>
  <zorder>lineEditLsyncPw</zorder>
  <zorder>lineEditVresHigh</zorder>
  <zorder>pushButtonFpaSet</zorder>
  <zorder>pushButtonFpaGet</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>lineEditVcsh</zorder>
  <zorder>label_10</zorder>
  <zorder>lineEditVcsl</zorder>
  <zorder>label_11</zorder>
  <zorder>lineEditVref</zorder>
  <zorder>label_12</zorder>
  <zorder>lineEditVcm1</zorder>
  <zorder>label_13</zorder>
  <zorder>lineEditVcm2</zorder>
  <zorder>label_14</zorder>
  <zorder>lineEditVcm3</zorder>
  <zorder>pushButtonFpaTemperatureGet</zorder>
  <zorder>lineEditFpaTemperature</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
